CSS/style.css

```diff
@@ -186,3 +186,495 @@ li a {
+  font-weight: 500;
+  left: -20px;
+}
+
+.used1-fr {
+  margin-left: -30px;
+  margin-top: 25px;
+}
+
+#recent-btn {
+  margin-left: 0;
+  margin-top: 30px;
+  position: static;
+  width: 120px;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+#recent-btn:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  box-sizing: border-box;
+}
+
+.work-btn {
+  align-items: center;
+  margin-top: -53px;
+  margin-left: -190px;
+  display: flex;
+}
+
+.see-btn {
+  padding-right: 217px;
+  padding-left: 140px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.see-btn:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  box-sizing: border-box;
+}
+
+.card-work {
+  display: grid;
+  margin-left: 24px;
+  padding-top: 24px;
+  grid-gap: 24px;
+  padding-bottom: 24px;
+  margin-right: 24px;
+  grid-template-rows: repeat(5, 357px);
+}
+
+.card1-work {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%);
+  width: 357px;
+  height: 386px;
+}
+
+.card1-work a img {
+  padding-bottom: 2px;
+}
+
+.card1-work-content {
+  position: relative;
+  top: -330px;
+}
+
+.work-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  color: #fbf8f7;
+  margin: 0 16px 0 16px;
+  position: static;
+  width: 295px;
+  height: 81px;
+  left: 16px;
+  top: 8px;
+  flex: none;
+  font-size: 36px;
+  line-height: 44px;
+}
+
+.card1-work-content p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  margin: 18px 16px 0 16px;
+  font-size: 18px;
+  line-height: 24px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  color: #fbf8f7;
+}
+
+.used1 {
+  background-color: rgb(78, 76, 76);
+  padding: 10px;
+  color: #fbf8f7;
+  margin-left: 8px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  left: -20px;
+}
+
+.main-about {
+  background-color: black;
+  margin-top: 80px;
+  height: 1700px;
+}
+
+#abt-title {
+  position: static;
+  width: 327px;
+  height: 79px;
+  margin: 0 24px 0 24px;
+  padding-top: 120px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 20px;
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+}
+
+#abt-text {
+  position: static;
+  width: 353px;
+  height: 148px;
+  margin: 12px 24px 24px 24px;
+  top: 223px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 26px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+#abt-frame {
```

Comment:
- [ ] Lets' not use id as a selector. IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse. It's much preferred to use classes in selectors and then apply a class to an element in the page. Additionally, IDs impact your specificity and can lead to specificity wars.